Dockerfile.telco-core

```diff
@@ -8,3 +10,15 @@ RUN microdnf install -y tar && \
 USER 65532:65532
 ENTRYPOINT ["/bin/bash"]
 CMD ["-c", "tar -cf - --directory /usr/share telco-core-rds | base64 -w0"]
+
+LABEL com.redhat.component="openshift-telco-core-rds-container" \
+      name="openshift4/openshift-telco-core-rds-rhel9" \
+      summary="Telco Core RDS manifests" \
+      io.openshift.expose-services="" \
+      io.openshift.tags="" \
+      io.k8s.display-name="openshift-telco-core-rds" \
+      maintainer="cnf-devel@redhat.com" \
+      description="Telco Core RDS manifests" \
+      io.openshift.maintainer.component="Telco Core RDS" \
+      io.openshift.maintainer.product="OpenShift Container Platform" \
+      io.k8s.description="Telco Core RDS manifests"
```

Comment:
Please fix the newline at the end of the file so future reviews don't show this as missing.